hw/xbox/nv2a/nv2a_debug.c

```diff
@@ -22,18 +22,19 @@
 
 #ifdef DEBUG_NV2A_GL
 
-#include <stdio.h>
-#include <stdbool.h>
-#include <stdarg.h>
 #include <assert.h>
+#include <stdarg.h>
+#include <stdbool.h>
+#include <stdio.h>
 
-static bool has_GL_GREMEDY_frame_terminator = false;
-static bool has_GL_KHR_debug = false;
+static bool has_GL_GREMEDY_frame_terminator;
+static bool has_GL_KHR_debug;
 
 void gl_debug_initialize(void)
 {
     has_GL_KHR_debug = glo_check_extension("GL_KHR_debug");
-    has_GL_GREMEDY_frame_terminator = glo_check_extension("GL_GREMEDY_frame_terminator");
+    has_GL_GREMEDY_frame_terminator = glo_check_extension(
+        "GL_GREMEDY_frame_terminator");
```

Comment:
Looks weird, having a string on same line is not bad in my opinion.

Reply:
Agree, but it got wrapped by clang-format because of the 80 character line limit rule.